typescript/web/src/components/labeling-tool/view-tool-bar/view-tool-bar.tsx

```diff
@@ -11,7 +11,7 @@ export const ViewToolbar = ({
 }) => {
   return (
     <>
-      <HideTool />
+      <ViewModeButton />
```

Comment:
Should be named `ViewModeTool` as the rest of the components in this page.

Reply:
Actually the whole `*-tool-bar` directories and their content are not that well named IMO.
But it's no problem, I'll name it ViewModeTool for now as you suggested and we'll do a more-general cleanup later.